astropy/modeling/tests/test_constraints.py

```diff
@@ -470,7 +485,10 @@ def test_gaussian2d_positive_stddev():
         [46.61, -104.11, 56.71, -90.85, -16.51, -66.45, -141.34, 0.96, 58.08,
          285.29, -61.41, -9.01, -323.38, 58.35, 80.14, -101.22, 145.65]]
     g_init = models.Gaussian2D(x_mean=8, y_mean=8)
-    fitter = fitting.LevMarLSQFitter()
+    if fitter == fitting.TRFLSQFitter:
+        pytest.xfail("TRFLSQFitter seems to be broken for this test.")
+    else:
+        fitter = fitter()
```

Comment:
@krachyon, again this test fails with the `TRFLSQFitter`, but I have not investigated why it fails yet.